### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by ChandlerAx** [cockpit-project/cockpit#20427] [General]
### Explain what happens

1. How to get the last version 316? when i try install cockpit i get 200 - 260
2. whats the best way to do this, thanks.
3. 

### Version of Cockpit

215

### Where is the problem in Cockpit?

Software updates

### Server operating system

Ubuntu

### Server operating system version

20.04.6 LTS

### What browsers are you using?

Edge

**Comment by jelly** [comment]
See our [official installation instructions](https://cockpit-project.org/running.html#ubuntu) they mention backports.

**Reply by ChandlerAx** [reply]
I dont understand the way backports work and some people told me it still isnt the last version, is this true?

**Comment by Bernd-Wissler-KBR** [comment]
I have cockpit 311 installed!
![image](https://github.com/cockpit-project/cockpit/assets/124667642/dee0d897-c9b6-4f9e-8a0d-3c0682c5ba54)

I want to have the newest release **317**, but "sudo apt install -t bookworm-backports cockpit" **says** that cockpit hast the newest version (311-1~bpo12+1)
![image](https://github.com/cockpit-project/cockpit/assets/124667642/f883213e-2ac7-4c70-b719-56aee7336b34)

How can i upgrade to the newest version?
